src/js/providers/providers-loaded.js

```diff
@@ -1,12 +1,11 @@
 define([
-    'providers/html5',
-    'providers/flash'
-], function(html5, flash) {
+    //'providers/html5',
```

Comment:
I'll add a comment. Any providers that we want bundled in jwplayer.js should be required there.